java/plantuml/Portfile

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                plantuml
+version             1.2018.0
+revision            1
```

Comment:
No need to set a revision for a new port. Just leave this out to use the default of `revision 0`.

Reply:
Ok, i'll fix it.